com.unity.render-pipelines.high-definition/Runtime/Lighting/LightLoop/DeferredTile.shader

```diff
@@ -43,7 +43,8 @@ Shader "Hidden/HDRP/DeferredTile"
 
             #define USE_INDIRECT    // otherwise TileVariantToFeatureFlags() will not be defined in Lit.hlsl!!!
 
-            #define USE_FPTL_LIGHTLIST 1 // deferred opaque always use FPTL
+            // Comment out the line to loop over all lights (for debugging purposes)
```

Comment:
is deferred tile working?

Reply:
Yes, you can test it yourself. Well, at least it worked before I switched to RG...